rules/aep0131/request_path_reference.go

```diff
@@ -20,10 +20,11 @@ import (
 	"github.com/jhump/protoreflect/desc"
 )
 
-var requestNameReference = &lint.FieldRule{
-	Name: lint.NewRuleName(131, "request-name-reference"),
+var requestPathReference = &lint.FieldRule{
+	Name: lint.NewRuleName(131, "request-path-reference"),
 	OnlyIf: func(f *desc.FieldDescriptor) bool {
-		return utils.IsGetRequestMessage(f.GetOwner()) && f.GetName() == "name"
+		return utils.IsGetRequestMessage(f.GetOwner()) && f.GetName() == "path"
 	},
 	LintField: utils.LintFieldResourceReference,
+	RuleType: lint.NewRuleType(lint.ShouldRule),
```

Comment:
not a must? seems like we should annotated this with a resource ref if possible.

Reply:
We specifically changed this to a should in the AEPs. Not sure why.